### PR TITLE
allocated resources shown in status should filter out not-yet-started and terminal containers

### PR DESCRIPTION
**Pull request by jdef**

### CODE REVIEW

**Comment by jdef** [comment]
retest this please

**Comment by jdef** [comment]
retest this please

**Comment by jdef** [comment]
@aquamatthias PTAL

**Comment by jdef** [comment]
Lots of this in the "package" build log. full disk?
```
[warn] Error extracting zip entry 'scala/util/Try$WithFilter.class' to '/jenkins/workspace/marathon-snapshot-packages-pulls/marathon-pkg/marathon/target/streams/$global/assemblyOption/$global/streams/assembly/1d2eafa8046aa70355212c53bfa5cfde1064ecc1_ddd5a8bced249bedd86fb4578a39b9fb71480573_da39a3ee5e6b4b0d3255bfef95601890afd80709/scala/util/Try$WithFilter.class': java.lang.RuntimeException: Could not create directory /jenkins/workspace/marathon-snapshot-packages-pulls/marathon-pkg/marathon/target/streams/$global/assemblyOption/$global/streams/assembly/1d2eafa8046aa70355212c53bfa5cfde1064ecc1_ddd5a8bced249bedd86fb4578a39b9fb71480573_da39a3ee5e6b4b0d3255bfef95601890afd80709/scala/util
```

File: src/main/scala/mesosphere/marathon/raml/PodStatusConversion.scala [hunk 1]
```diff
@@ -238,6 +225,29 @@ trait PodStatusConversion {
...
   }
+
+  /**
+    * @return the resources actually allocated/in-use by this pod instance; terminal instances don't count
+    */
+  def allocatedResources(pod: PodDefinition, instance: Instance): Option[Resources] = {
+    import InstanceStatus._
+
+    instance.state.status match {
+      case Staging | Starting | Running | Reserved | Unreachable | Killing =>
```
Comment (aquamatthias) [inline src/main/scala/mesosphere/marathon/raml/PodStatusConversion.scala]: There are several instance states, that can differ from task states: e.g the instance will go into state Failed, if one task fails. The whole instance still would consume resources.

A high coupling is reflected here between the instance state and the task state, that we probably should avoid.

2 options here:
1) you return always `Resources` where the actual values are 0
2) you maintain an `Option[Resources]`, if resources are 0

Comment (jdef) [inline src/main/scala/mesosphere/marathon/raml/PodStatusConversion.scala]: ack

**Comment by jdef** [comment]
retest this please

**Comment by jdef** [comment]
retest this please

**Comment by aquamatthias** [comment]
Thanks!

**Pull request merged**